### PR TITLE
Fix 9

### PR DESCRIPTION
**Pull request by rsjouwerman**
Please check if she is gone

### CODE REVIEW

**Comment by mana-ehlers** [comment]
She is really gone.

File: games2.Rmd [hunk 1]
```diff
@@ -16,6 +16,8 @@ conflicted::conflict_prefer("filter", "dplyr")
 ```
 
 ## Acknowledgement
+Lisa Simpson is not here anymore! 
+
```
Comment (alkastrinogiannis) [inline games2.Rmd:20]: Where is she?

**Review by alkastrinogiannis — COMMENTED** [review]
Nice changes!

**Review by alkoppold — COMMENTED** [review]
wow

File: games2.Rmd (hunk 1 above)
Comment (tinalonsdorf) [reply games2.Rmd:20]: gone!

Comment (rsjouwerman) [reply games2.Rmd:20]: she's on vacation

**Review by tinalonsdorf — APPROVED** [review]

**Review by alkoppold — APPROVED** [review]

File: games2.Rmd (hunk 1 above)
Comment (m-klingelhoefer-j) [reply games2.Rmd:20]: I don't know!

**Review by mana-ehlers — APPROVED** [review]

**Review by m-klingelhoefer-j — APPROVED** [review]

**Pull request merged**